packages/discord.js/typings/index.d.ts

```diff
@@ -398,28 +398,6 @@ export type GuildCacheMessage<Cached extends CacheType> = CacheTypeReducer<
   Message | APIMessage
 >;
 
-export interface InteractionResponseFields<Cached extends CacheType = CacheType> {
```

Comment:
Why this was deleted?

Reply:
https://github.com/discordjs/discord.js/pull/8565#pullrequestreview-1087841227

Reply:
Oh, got it